File: Data Structures & Algorithms/longest-common-prefix/submission-1.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        temp = strs[0]
        for i in range(1, len(strs)):
            j = 0
            while(j < min(len(strs[i]), len(temp))):
                if temp[j] != strs[i][j]:
                    break
                j = j+1
            temp = temp[:j]
        return temp     

File: Data Structures & Algorithms/arranging-coins/submission-1.java
class Solution {
    public int arrangeCoins(int n) {
        int len = 0;
        int i = 1;
        while(n >= i){
            len++;
            n -= i;
            System.out.println((n+i)+" "+n);
            i++;
        }
        return len;
        
    }
}